EIC-EA/EIC-community-D8
# add error handler and message on VOD and quality selector

https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EICNET-2325

### What I did 

- add quality selector on VOD player
- add error block when the video is on process 

### How to test

- go to add video 
- add video file ( not an URL)
- and see the result
